CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.1.0dev
```

Comment:
will be updated by https://github.com/nf-core/hic/pull/168

Reply:
This is the typical thing I forget to do before releasing 🤣

Reply:
I either do 2 PRs, or do it all in one